utils/reportsender.go

```diff
@@ -0,0 +1,62 @@
+package utils
```

Comment:
I think you can remove this file

Reply:
Good catch! That was left over from the merge.

Reply:
Fixed.